test/BabelRewirePluginTransformTest.js

```diff
@@ -28,7 +28,13 @@ describe('BabelRewirePluginTest', function() {
 			expect().fail("Transformation failed: \n" + error.stack)
 		}
 
-		fs.writeFileSync('/tmp/testexpected' + testName + '.js', transformationOutput, 'utf-8');
+		var tempDir = '/tmp/babel-plugin-rewire';
```

Comment:
@TheSavior I agree. The original usage of /tmp was a hack and not supposed to stay in the code ;).